ts/src/test/static/request/woo.json

```diff
@@ -387,12 +387,64 @@
         ],
         "fetchDepositAddress": [
             {
-                "description": "fetch usdt address",
+                "description": "coin with primary network",
```

Comment:
added all 4 cases of combinations:
- primary coin & primary network (eg. ETH & ETH)
- primary coin & secondary network (eg. ETH & ERC20)
- token & primary network (eg. USDT & ETH)
- token & secondary network (eg. USDT & ERC20)